### DISCUSSION TITLE
Can't renew subscription

### DISCUSSION THREAD

**Discussion by omgpiu** [community/community#61455] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

When my subscription has ended I can't again pay from my card for subscription. I has to add a new card every time? I got an error - Your account is currently locked from purchases. Please update your payment information.

Can you fix this annoying bug? Second time I has to change my card. Even if i delete card and try to add this card with money, it doesn't work.

**Comment by Akash1134** [comment, marked as answer]
Hi @omgpiu  The best place to go for answers to account-specific and billing related questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
